src/utils.ts

```diff
@@ -3,9 +3,27 @@ import logger from 'logger';
 import Settings from 'services/settings.service';
 import { IUser } from 'services/okta.interfaces';
 import { URL } from 'url';
+import { PaginateOptions } from 'mongoose';
 
 export default class Utils {
 
+    static getPaginationParameters(ctx: Context): PaginateOptions {
+        let page: number = 1;
+        let limit: number = 10;
+
+        if (ctx.query.page) {
+            // tslint:disable-next-line:variable-name
+            const { number, size } = (ctx.query.page as Record<string, any>);
+            page = number ? parseInt(number, 10) : 1;
```

Comment:
Minor but  `if (number) page = parseInt(number, 10)` would avoid having to re-define defaults defined above.